src/allencell_ml_segmenter/training/view.py

```diff
@@ -296,18 +297,46 @@ def getTypeOfWork(self) -> str:
         return "Training"
 
     def showResults(self) -> None:
-        csv_path: Optional[Path] = (
-            self._experiments_model.get_latest_metrics_csv_path()
+        # double check to see if a ckpt was generated
+        exp_path: Optional[Path] = (
+            self._experiments_model.get_user_experiments_path()
         )
-        if csv_path is None:
-            raise RuntimeError("Cannot get min loss from undefined csv")
-        min_loss: Optional[float] = FileUtils.get_min_loss_from_csv(csv_path)
-        if min_loss is None:
-            raise RuntimeError("Cannot compute min loss")
-        dialog_box = InfoDialogBox(
-            "Training finished -- Final loss: {:.3f}".format(min_loss)
+        if exp_path is None:
+            raise ValueError(
+                "Experiments path should not be None after training complete."
+            )
+        exp_name: Optional[str] = self._experiments_model.get_experiment_name()
+        if exp_name is None:
+            raise ValueError(
+                "Experiment name should not be None after training complete."
+            )
+        ckpt_generated: Optional[Path] = ExperimentUtils.get_best_ckpt(
+            exp_path,
+            exp_name,
         )
-        dialog_box.exec()
+        if ckpt_generated is not None:
+            # if model was successfully trained, get metrics to display
+            csv_path: Optional[Path] = (
+                self._experiments_model.get_latest_metrics_csv_path()
+            )
+            if csv_path is None:
+                raise RuntimeError("Cannot get min loss from undefined csv")
+            min_loss: Optional[float] = FileUtils.get_min_loss_from_csv(
+                csv_path
+            )
+            if min_loss is None:
+                raise RuntimeError("Cannot compute min loss")
+
+            dialog_box = InfoDialogBox(
+                "Training finished -- Final loss: {:.3f}".format(min_loss)
+            )
+            dialog_box.exec()
+            self._main_model.training_complete()
```

Comment:
I wonder about tying this event to the show results dialog, rather than just rely on a single TRAINING_COMPLETE event for both showing the dialog and changing the tab.

Reply:
So use this even to show the result dialog? Instead of showing it in the training UI code and emitting the event afterwards?

Reply:
Ok.  Maybe we should rename showResults.
Also, for now just comment what `training_complete` is about

Reply:
https://github.com/AllenCell/allencell-segmenter-ml/pull/519/commits/d2f67edf42fa5ca9e3d8cff5fba62a3b62e5003b

Reply:
https://github.com/AllenCell/allencell-segmenter-ml/issues/521